hnn_core/cells_default.py

```diff
@@ -146,6 +146,11 @@ def _get_mechanisms(p_all, cell_type, section_names, mechanisms):
     return mech_props
 
 
+def _set_variable_mech(dist_from_soma):
+    """Set a cell mechanism based on its distance from the soma"""
+    return 1e-6 * np.exp(3e-3 * dist_from_soma)
+
+
 def basket(pos, cell_name='L2Basket', gid=None):
```

Comment:
Does this default name get used in the code anywhere?

Reply:
I don't think so. Let's remove it unless @jasmainak objects?